index.html

```diff
@@ -245,6 +245,28 @@
   </section>
 
   <section>
+    <h2>Security and Privacy</h2>
+
+    <p>The use of <a href="#dfn-resource-hint-link">resource hints</a> can provide new means to expose information about both the current page and user's activity on it. For example, observing the content of prefetched or prerendered resources, or observing the DNS, IP, and hostname information exposed during connection establishment, both for secure and plain-text connections, can reveal information about the page initiating such requests, and where dynamic resource hints are used, user's activity on it.</p>
+
```

Comment:
Note the "new means"... all of this is possible today by simply observing what resources the site is fetching and from which origins. I just want to be thorough in documenting the possible risks.